com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -92,24 +98,72 @@ void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
 
             (Camera camera, UniversalRenderPipelineSerializedCamera serializedCamera) overlayCamera = m_SerializedCamera[index];
             Camera cam = overlayCamera.camera;
+            var baseAdditionalData = camera.GetUniversalAdditionalCameraData();
```

Comment:
Can we move this underneath the `if (cam != null)` ?

Reply:
fixed